erpnext/stock/get_item_details.py

```diff
@@ -481,7 +481,7 @@ def get_projected_qty(item_code, warehouse):
 @frappe.whitelist()
 def get_bin_details(item_code, warehouse):
 	return frappe.db.get_value("Bin", {"item_code": item_code, "warehouse": warehouse},
-			["projected_qty", "actual_qty"], as_dict=True) \
+			["projected_qty", "actual_qty", "reserved_qty"], as_dict=True) \
 			or {"projected_qty": 0, "actual_qty": 0}
```

Comment:
I guess, reserved_qty should have default value as well.
{"projected_qty": 0, "actual_qty": 0, "reserved_qty":0}